pipeline/azure-pipelines.yml

```diff
@@ -0,0 +1,244 @@
+      includeNuGetOrg: false
+      nugetConfigPath: nuget.config
+      arguments: restore -LockedMode -Verbosity detailed
+
+  - task: NuGetCommand@2
+    displayName: NuGet restore without LockedMode check (VS2019)
+    condition: and(succeeded(), ne(variables['vsTargetVersion'], '2022'))
+    inputs:
+      solution: $(BuildParameters.solution)
+      selectOrConfig: config
+
+  - task: PowerShell@2
+    displayName: 'Read Sonar project version from the versions.props file '
+    inputs:
+      targetType: filePath
+      filePath: $(System.DefaultWorkingDirectory)\pipeline\scripts\read-versions.props.ps1
+
+  - task: PowerShell@2
+    displayName: Locate signtool.exe
+    inputs:
+      targetType: filePath
+      filePath: $(System.DefaultWorkingDirectory)\pipeline\scripts\locate-signtool.ps1
+
+  - task: PowerShell@2
+    displayName: (VS2022 only) Set VsixVersion as a build property for the release pipeline to use
+    env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+    condition: and(succeeded(), eq(variables['vsTargetVersion'], '2022'))
+    inputs:
+      targetType: filePath
+      filePath: $(System.DefaultWorkingDirectory)\pipeline\scripts\set-vsix-version.ps1
+
+  - task: PowerShell@2
+    displayName: (VS2022 only) Sanity check - fetch VsixVersion build property
+    condition: and(succeeded(), eq(variables['vsTargetVersion'], '2022'))
+    inputs:
+      targetType: filePath
+      filePath: $(System.DefaultWorkingDirectory)\pipeline\scripts\fetch-vsix-version.ps1
+
+  - task: VSBuild@1
+    displayName: Set BranchName, Sha1 and BuildNumber properties from Azdo pipeline variables
+    inputs:
+      solution: build/ChangeVersion.proj
+      msbuildArgs: /p:Sha1=$(Build.SourceVersion) /p:BranchName=$(Build.SourceBranchName) /p:BuildNumber=$(Build.BuildId)  /p:BuildConfiguration=$(BuildConfiguration)
+
+  - task: DownloadSecureFile@1
+    name: snk
+    displayName: Download snk file
+    inputs:
+      secureFile: c638339f-2dc0-42e1-9b9c-7ee76eda15f5
+      retryCount: 5
+
+  - task: DownloadSecureFile@1
+    name: pfx
+    displayName: Download pfx file
+    inputs:
+      secureFile: c2ebf861-df45-46e0-9245-6ef7ae954a8b
+      retryCount: 5
+
+  - task: SonarCloudPrepare@1
+    displayName: Prepare analysis on SonarCloud (VS2022 only)
+    condition: and(succeeded(), eq(variables['vsTargetVersion'], '2022'))
+    inputs:
+      SonarCloud: 4d6d3285-de8e-41bb-a346-987397025edb
+      organization: sonarsource
+      projectKey: sonarlint-visualstudio
+      projectName: SonarLint for Visual Studio
+      projectVersion: $(SONAR_PROJECT_VERSION)
+      extraProperties: "# Additional properties that will be passed to the scanner, \n# Put one key=value per line, example:\n# sonar.exclusions=**/*.bin\n#sonar.cs.vstest.reportsPaths=$(Agent.TempDirectory)/**/*.trx\n#sonar.cs.vscoveragexml.reportsPaths=$(Agent.TempDirectory)/**/*.coveragexml"
```

Comment:
This doesn't look right - everything is on one line.
However, I don't think it matters since all of the extra properties are commented out. If that is correct then it would be better to just remove the `extraProperities`.

Reply:
Again this is the result of export. And Information is copied from the job. If this would not work we can remove I guess. 

![image](https://user-images.githubusercontent.com/96827714/194855944-a09f3d16-2dca-4b9c-9b5c-84894e09c597.png)